interop/src/clients/InteropClient/delete_simulator.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+
+DEVICE=$1
+
+xcrun simctl shutdown $DEVICE
+xcrun simctl delete $DEVICE
```

Comment:
This will take care of the leftover processes issue we have, right?

Reply:
Yes